### DISCUSSION TITLE
Practica - Toma de requerimientos - Descripción solicitud - Líder funcional

### DISCUSSION THREAD

**Discussion by RicardoFT1** [Launch-X-Latam/MisionFrontEnd#45] [Q&A]
¡Hola Explorers!

1- Desconozco completamente la Netiquetá de estos chats, como primera instancia debo consultar para no pasar por grosero es si ¿se debe pasar la pregunta directa al grano sin saludos y cordialidades, o es más prudente siempre iniciar con un saludo y demás?

2- En la Toma de requerimientos - Descripción solicitud - Líder funcional, ¿es posible de una vez tener en cuenta posibles mejoras en el requerimiento del cliente?.

Ejemplo: El cliente pide que sus usuarios creen una cuenta para ver las actualizaciones del proceso legal, yo ofrecería una vez reconocido el pago crear la cuenta con los datos del formulario inicial y enviarla por correo electrónico con los correspondientes procesos de seguridad. 

Gracias.

Cordialmente.

Ricardo F.

**Comment by fedeminatta** [comment]
tenia entendido q mañana arrancabamos con las practicas creo

**Reply by RicardoFT1** [reply]
Hola, si, es muy cierto que inician mañana, sin embargo en mi opinión entiendo que por ser un Bootcamp autodidacta hay que indagar e investigar todo el material que habilitaron, así cuando lleguemos a la siguiente sesión al ser esta de un corto tiempo podremos estar mas en contexto al momento que aborden el tema.
